_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,346 @@
+
+## Guiding Policy
+
+The following policy reflects the decision adopted by the **CIO of ESDC** (approval by CIO not yet obtained) regarding usage of the Version Control Systems.
+Each policy statement is a declaration of that decision and has received the endorsement of its associated area of governance body (endorsements not yet obtained, see [Coherent set of actions](#coherent-set-of-actions)).
+
+This policy becomes active when IT Products are to be built.
+Once active, all teams involved in the project, and the IT products involved in the IT solution, must comply with this guiding policy.
+
+This Guiding Policy has been prepared by taking into consideration alignment and compliance with existing policy instruments and does not replace them.
+Stakeholders are expected to still comply with existing policy instruments including, but not limited to:
+
+- TBS [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601)
+- TBS [Directive on Security Management](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32611)
+- ESDC Information Management Policy (being drafted)
+- ESDC Security Policy (being drafted)
+
+### Development and Operations Teams (incl. SSC) <!-- omit in toc -->
+
+1. Use a Git-based Version Control System (VCS) among the following options, in that order of preference:
+   - Public (Unclassified): **[GitHub](https://github.com)** or another [approved public platform (internal)](https://dialogue/grp/TAWS-ATST/Technology_Standards/Brick%20Repository/Software%20Version%20and%20Source%20Code%20Management.mht)
+   - Interdepartmental (up to Protected A): **[GCcode (internal)](https://gccode.ssc-spc.gc.ca)**
+   - Internal (up to Protected B): **[Azure DevOps](https://azure.microsoft.com/en-ca/services/devops/)**
+2. Follow the guidance of [ESDC Open Source Software (OSS) Management Framework (internal)](https://dialogue/grp/TAWS-ATST/Collaboration%20Architecture%20Services/Projects/Artificial%20Intelligence%20and%20Open%20Source/OSS/ESDC%20Open%20Source%20Management%20Framework%20v%201.1.docx).
+3. Leverage the functionalities provided by the selected VCS as much as possible, for instance:
+    - File hosting (source code, scripts, system/database configurations...);
+    - Issues (incl. milestones and Kanban);
+    - Wiki (documentation);
+    - Build and Test Automation (e.g., security-related); and
+    - Deployment to internal or external platforms.
+4. Make their projects in the VCS open by default.
+
+## Coherent set of actions
+
+The following are actions that need to be performed in order to make the Version Control Systems operational.
+
+<table>
+  <tr>
+    <th>Outcome</th>
+    <th>Action</th>
+    <th>Description</th>
+    <th>Lead</th>
+    <th>Contributor(s)</th>
+  </tr>
+  <tr>
+    <td rowspan="3"><b>Architecture</b></td>
+    <td>Update ESDC OSS Management Framework</td>
+    <td>Update ESDC OSS Management Framework to reflect the expected usage and capabilities of each approved VCS</td>
```

Comment:
Should the OSS framework be targeted here? Or should we avoid adding delays for its endorsement?

Reply:
Good point! I've now put the Endorsement before the Update so that it doesn't add delays to the endorsement already in progress.

Reply:
I mean, not done "now" but the commit is upcoming :)